### PR TITLE
UC29: Forgot password

### PR DESCRIPTION
**Pull request by Kwistech**
Refactored backend to function with the following request:

Capture

### CODE REVIEW

**Comment by matthewjflee** [comment]
What does the email look like?

File: src/main/java/ca/sait/vezorla/controller/ClientRestController.java [hunk 1]
```diff
@@ -43,6 +45,14 @@ public boolean updateAccount(@RequestBody Account account, HttpServletRequest re
         return created;
     }
 
+
+    @GetMapping("account/forgotpassword")
```
Comment (matthewjflee) [inline src/main/java/ca/sait/vezorla/controller/ClientRestController.java]: `@PutMapping`

Comment (Kwistech) [reply src/main/java/ca/sait/vezorla/controller/ClientRestController.java]: Yes, we are altering the database. The change does work as we want to return a boolean to the frontend. I will make the change shortly.

**Comment by Kwistech** [comment]
Steps taken to correct functionality:
1. Port 587 
- Antivirus blocked port
- Firewall may have blocked port
- SAIT laptop policy may have blocked port
- My router disabled port
- ISP may be blocking port
Solution = Use a VPN

2. Gmail account
- Bad credentials (created new Gmail account to test)
- Activate App Password
- Activate 2FA
- Set password to App password
- Enable insecure apps to connect
- Connection successful, email sent, password reset.

Thoughts?

Capture

Capture2

**Pull request merged**